### DISCUSSION TITLE
Change logs for user actions?

### DISCUSSION THREAD

**Discussion by fjallemark** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#101] [Ideas]
Some users are more concerned with the data quality than others, we can call them *super users*. In order to monitor changes in data for modules and stations it is possible to create a *change log* for modules and stations.

In the *change log*, it will be possible to see who changed what for the specific module or station and when. The *change log* would be accessible from each module and station.

Is this something that will be useful to *super users*?

**Comment by turvik** [comment]
A bit late to the party here, but I would say Yes!
